puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/rpm_packaging.groovy

```diff
@@ -65,22 +65,25 @@ pipeline {
 
                     for(int i = 0; i < packages_to_build.size(); i++) {
                         package_name = packages_to_build[i]
+                        spec_pattern = "packages/**/${package_name}.spec"
 
-                        sh "git checkout origin/${env.ghprbTargetBranch}"
+                        old_spec_path = find_deleted_files("origin/${env.ghprbTargetBranch}", spec_pattern)
+                        if (! old_spec_path) {
+                            old_spec_path = find_added_or_changed_files("origin/${env.ghprbTargetBranch}", spec_pattern)
+                        }
```

Comment:
Looks like you could drop the `if (old_spec_path) {` that follows this line and change it to an `else`

Reply:
No? I want:
```
old_spec_path = find_delted_files() || find_added_or_changed_files()
if (old_spec_path) …
```

Your suggestion would drop the `|| find_added_or_deleted` part?